test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
Looks like rspec is not completely integrates in the app (it should be `spec/models/user_spec.rb`

Reply:
We also need to have specs for validations you added. Please use shoulda matchers for that